visivo/templates/render_yaml.py

```diff
@@ -0,0 +1,109 @@
+import time
+import datetime
+from dateutil import parser
+import jinja2
+import os
+
+
+def env_var(key):
+    return os.getenv(key, "NOT-SET").replace("\\", "\\\\").replace('"', '\\"')
```

Comment:
Where was this?  I think it should be `${key}-ENVIRONMENT-VARIABLE-NOT-SET` if possible.

Reply:
I moved this function over as it was from utils.py. I can definitely update it to  say that instead.